src/_help/getting-started/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Quick Start
+---
+
+- TOC
+{:toc}
+
+Bump.sh connects people with REST and event-driven APIs, by centralizing a reference point: their documentation, highlighted with the latest changes and updates.
+
+From your API definition (OpenAPI and AsyncAPI), Bump.sh automatically generates the documentation you need, a detailed changelog of the latest changes, and offers you many options to customize it to provide a unique experience that is your own.
+
+This Getting Started guide aims to get you started quickly if you're writing OpenAPI or AsyncAPI files or if you know where these files are located in your code repository.
+
+You will start by [deploying your first API documentation](/getting-started/upload-your-first-definition/) and then explore initial customization options before [delving deeper](/going-further/).
```

Comment:
```suggestion
You will start by [deploying your first API documentation](/getting-started/upload-your-first-definition/) and then explore initial customization options before [digging further](/going-further/).
```

Although I enjoy that vocabulary, I would opt for more straightforward/"more international" English.